### DISCUSSION TITLE
mini-indentscope: is there a way to have the indent scoping as straight lines with no spaces in between?

### DISCUSSION THREAD

**Discussion by idelice** [nvim-mini/mini.nvim#892] [General]
title

**Comment by echasnovski** [comment]
Set appropriate [`config.symbol`](https://github.com/echasnovski/mini.nvim/blob/cd3a08fbf987dabcfded35ebb612e82f8000f92a/doc/mini-indentscope.txt#L164). For example, like this:

```lua
require('mini.indentscope').setup({ symbol = '│' })
```

Here is a [good reference for other symbols](https://en.wikipedia.org/wiki/Box-drawing_characters). Also note that their visual appearance depends on how terminal emulator displays them (so not 100% that they will have no space in between cells).

**Reply by idelice** [reply]
Amazing. Thank you

**Reply by idelice** [reply]
I'm using kitty and it works perfectly fine!
